extension/Config.jsm

```diff
@@ -0,0 +1,69 @@
+/* to use:
+
+- Recall this file has chrome privileges
+- Cu.import in this file will work for any 'general firefox things' (Services,etc)
+  but NOT for addon-specific libs
+*/
+
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(config|EXPORTED_SYMBOLS)" }]*/
+const EXPORTED_SYMBOLS = ["config"];
+
+// var slug = "shield-example-addon"; // should match chrome.manifest;
+
+const config = {
+  study: {
+    studyName: "share-button-study", // no spaces, for all the reasons
+    variation: {
+      name: "ALL",
+    }, // optional, use to override/decide
+    weightedVariations: [
+      { name: "control", weight: 1 },
+      { name: "DOORHANGER", weight: 2 },
+      { name: "HIGHLIGHT", weight: 0.5 },
```

Comment:
In this case, the weights are arbitrary as I am setting the variation to `ALL` explicitly.

The reason for all-caps is because this matches the treatment names in my `bootstrap.js` code, but this is not strictly necessary.